configs/projects/agent-runtime-1.10.x.rb

```diff
@@ -57,4 +57,5 @@
 
   proj.component 'rubygem-gettext-setup'
   proj.component 'ruby-stomp'
+  proj.component 'rubygem-win32-eventlog' if platform.is_windows?
```

Comment:
Is this necessary?

Reply:
I'm not really familiar with what is still needed on 1.10.x, but since it won't be around for that much we can let this here to not brake anything.